walkthroughs/rhte-2019-hackathon/walkthrough.adoc

```diff
@@ -559,15 +553,19 @@ Use the following details to connect:
 
 Details of table names and schemas were provided in the previous section - *API Definition using Apicurio*
 
-[time=60]
-== API Implementation using Node.js
+[time=90]
+== API Implementation using Camel + Spring Boot or Node.js
+
+=== Camel + Spring Boot
+
```

Comment:
@hguerrero - have a placeholder here for the Camel + Spring boot walkthrough instruction. Can you update with details similar to what Evan has added for the Node.js one.